.github/workflows/build.yml

```diff
@@ -48,4 +48,18 @@ jobs:
         run: |
           make buildx.image
           make buildx.push
-
+        
+  trivy:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@master
+        with:
+          image-ref: 'openebs/linux-utils:ci'
```

Comment:
```suggestion
        uses: actions/checkout@v2
        
      - name: Set tag
        run: |
          BRANCH="${GITHUB_REF##*/}"
          CI_TAG=${BRANCH#v}-ci
          if [ ${BRANCH} = "master" ]; then
            CI_TAG="ci"
          fi
          echo "::set-env name=TAG::${CI_TAG}"
        
      - name: Run Trivy vulnerability scanner
        uses: aquasecurity/trivy-action@master
        with:
          image-ref: 'openebs/linux-utils:${{ env.CI_TAG }}'
```

@prakhargurunani One more fix needs to be done. Sorry  I missed this the first time.

We need this so that trivy can be run on non-master branches also.